CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
-# v24.0.0-snapshot
+# v24.2.0
```

Comment:
We will have a v25 client soon as Kubernetes 1.25 gets released. Do you think you can wait for that instead of having a patch release to v24?

cc @yliaog